op-service/eth/ssz_test.go

```diff
@@ -122,28 +204,131 @@ func TestOPB01(t *testing.T) {
 	copy(data[504:508], make([]byte, 4))
 
 	var unmarshalled ExecutionPayload
-	err = unmarshalled.UnmarshalSSZ(uint32(len(data)), bytes.NewReader(data))
+	err = unmarshalled.UnmarshalSSZ(BlockV1, uint32(len(data)), bytes.NewReader(data))
 	require.Equal(t, ErrBadTransactionOffset, err)
 }
 
 // TestOPB04 verifies that the SSZ marshaling code
 // properly returns an error when the ExtraData field
 // cannot be represented in the outputted SSZ.
 func TestOPB04(t *testing.T) {
-	// First, test the maximum len - which in this case is the max uint32
-	// minus the execution payload fixed part.
-	payload := &ExecutionPayload{
-		ExtraData: make([]byte, math.MaxUint32-executionPayloadFixedPart),
+	tests := []struct {
+		version     BlockVersion
+		withdrawals *Withdrawals
+	}{
+		{BlockV1, nil},
+		{BlockV2, &Withdrawals{}},
 	}
-	var buf bytes.Buffer
-	_, err := payload.MarshalSSZ(&buf)
-	require.NoError(t, err)
-	buf.Reset()
 
-	payload = &ExecutionPayload{
-		ExtraData: make([]byte, math.MaxUint32-executionPayloadFixedPart+1),
+	for _, test := range tests {
+		// First, test the maximum len - which in this case is the max uint32
```

Comment:
do you need `test := test`?

Reply:
I don't believe so, as we're not capturing it in anything that can run async.